src/pages/settings/AboutPage/AboutPage.js

```diff
@@ -116,29 +117,21 @@ const AboutPage = (props) => {
                             'initialSettingsPage.readTheTermsAndPrivacyPolicy.phrase1',
                         )}
                         {' '}
-                        <Text
-                            accessibilityRole="link"
-                            style={[styles.textMicroSupporting, styles.link]}
-                            onPress={() => Link.openExternalLink(CONST.TERMS_URL)}
-                        >
+                        <TextLink style={[styles.textMicroSupporting, styles.link]} href={CONST.TERMS_URL}>
```

Comment:
It fells like we should remove `styles.link` since it's already included in [TextLink](https://github.com/Expensify/App/blob/e434efa1c455620c510441628b94081e0f8a564e/src/components/TextLink.js#L61). However, doing so would break the styling, on this [line](https://github.com/Expensify/App/blob/e434efa1c455620c510441628b94081e0f8a564e/src/components/TextLink.js#L61) blue color from `styles.link` is overriden with grey color from `additionalStyles` (`styles.textMicroSupporting`)
We could just swap the order like this `style={[...additionalStyles, styles.link]}`, but I'm afraid it will break all of the other usages of `TextLink` throughout the app. 

@0xmiroslav, do you have ideas on how this can be resolved without introducing lots of changes?